tests/test_scalar_multiplication.py

```diff
@@ -1,6 +1,7 @@
 from math import isclose
 
-from hypothesis import assume, given
+from hypothesis import assume, given, strategies as st
+from pytest import raises  # type: ignore
```

Comment:
:sigh: what complains about this?

Reply:
`mypy` simply doesn't have an interface definition for `pytest`, which is why all `pytest` imports need to have a `type: ignore` annotation.